### PR TITLE
Fix compilation warnings

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

File: Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation2D.cxx [hunk 1]
```diff
@@ -148,7 +148,6 @@ void vtkSlicerAngleRepresentation2D::BuildArc()
   double vector2[3] = { p2[0] - c[0], p2[1] - c[1], p2[2] - c[2] };
   double l1 = vtkMath::Normalize(vector1);
   double l2 = vtkMath::Normalize(vector2);
-  double angle = acos(vtkMath::Dot(vector1, vector2));
```
Comment (jcfr) [inline Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation2D.cxx]: @lassoan  This is the only one worth checking, did you intended to use the angle variable ?

**Comment by jcfr** [comment]
@lassoan  I will move forward with the integration, I still suggest you review them if these were revealing actual issues.

**Comment by jcfr** [comment]
Closing. Change integrated in [r28307](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=28307) to [r28311](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=28311)

File: Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation2D.cxx (hunk 1 above)
Comment (lassoan) [reply Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation2D.cxx]: Angle measurement was moved to the MRML node, so it is not needed here anymore.

**Comment by lassoan** [comment]
Thank you!

**Comment by jcfr** [comment]
Great. Thanks for reviewing :+1:

**Pull request closed without merging**